frontend/compositions/visualizations/map/BaseMap.tsx

```diff
@@ -1,10 +1,6 @@
-import * as d3 from "d3"
+import { geoPath, scaleLinear, interpolate, select } from "d3"
 import { Feature, FeatureCollection } from "geojson"
 import React, { useEffect, useRef, useState } from "react"
-import * as topojson from "topojson-client"
-import { presimplify, simplify } from "topojson-simplify"
-import { Topology } from "topojson-specification"
-import { FakeData } from "../utilities/chartTypes"
 
 // colors:
```

Comment:
I see it's not your code, but if you don't mind scrapping the comments here too it'd make my heart happy 😅